openlibrary/admin/numbers.py

```diff
@@ -132,7 +131,7 @@ def admin_range__visitors(**kargs):
         sqlitefile = tempfile.mktemp(prefix="sqlite-")
         url = "http://www.archive.org/download/stats/numUniqueIPsOL.sqlite"
         logging.debug("  Downloading '%s'", url)
-        sqlite_contents = urllib.request.urlopen(url).read()
+        sqlite_contents = requests.get(url).content
```

Comment:
Two choices here (I am unsure which is best):
1. Str route: `requests.get(url).text` instead of `requests.get(url).content` -- or --
2. Bytes route: `f = open(sqlitefile, "wb")`instead of `f = open(sqlitefile, "w")`